content/en/docs/ops/common-problems/security-issues/index.md

```diff
@@ -79,42 +79,62 @@ With Istio, you can enable authentication for end users through [request authent
     <redacted>
     {{< /text >}}
 
-## Authorization is too restrictive
-
-When you first enable authorization for a service, all requests are denied by default. After you add one or more authorization policies, then
-matching requests should flow through. If all requests continue to be denied, you can try the following:
+## Authorization is too restrictive or permissive
+
+### Make sure there are no typo in the policy YAML file
+
+One common mistake is specifying unexpected multiple items in the YAML, take the following policy as an example:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: example
+  namespace: foo
+spec:
+  action: ALLOW
+  rules:
+  - to:
+    - operation:
+        paths:
+        - /foo
+  - from:
+    - source:
+        namespaces:
+        - foo
+{{< /text >}}
 
-1. Make sure there is no typo in your policy YAML file.
+You may expect the policy to allow requests if the path is `/foo` **and** the source namespace is `foo`.
+However, the policy actually allows requests if the path is `/foo` **or** the source namespace is `foo`, which is
+more permissive.
 
-1. Avoid enabling authorization for {{< gloss >}}Istiod{{< /gloss >}}. Istio authorization policy is designed for authorizing access to workloads in Istio Mesh. Enabling it for Istiod may cause unexpected behavior.
+In the YAML syntax, the `-` in front of the `from:` means it's a new element in the list. This creates 2 rules in the
+policy instead of 1. In authorization policy, multiple rules have the semantics of `OR`.
 
-1. Make sure that your authorization policies are in the right namespace (as specified in `metadata/namespace` field).
+To fix the problem, just remove the extra `-` to make the policy have only 1 rule that allows request if the
+path is `/foo` **and** the source namespace is `foo`, which is more restrictive.
 
-1. Make sure that your authorization policies with ALLOW action don't use any HTTP only fields for TCP traffic.
-Otherwise, Istio ignores the ALLOW policies as if they don't exist.
+### Make sure you are NOT using HTTP-only fields on TCP ports
 
-1. An HTTP response with the value `RBAC: Access Denied` indicates an authorization policy is in effect. You can determine the authorization policy in effect by running `istioctl x authz check POD-NAME.POD-NAMESPACE`.
+The authorization policy will be more restrictive because HTTP-only fields (e.g. `host`, `path`, `headers`, JWT, etc.)
+will never be matched on raw TCP connections. This could cause unexpected 403 denies.
```

Comment:
Does the inverse apply? That it may lead to an allow, because we do not match the deny policy?

Reply:
The policy always becomes more restrictive, updated to make it clear for each action.